README.md

```diff
@@ -27,5 +27,12 @@ A nodeJS youtube jukebox application. Great for parties and stuff :-)
 1. Speaker - `sudo apt-get install libasound2-dev`
 1. GCC Compiler - `sudo apt-get install gcc`
 
+### Mac OS X
+
+#### OS X version 10.11 and up
+1. FFmpeg - If you haven't yet, install [Homebrew](https://brew.sh), then run `brew install ffmpeg`.
+1. Speaker - If you haven't, install Node and npm using homebrew again --> `brew install node`. Then run `sudo npm install speaker`.

```

Comment:
Maybe remove the "if you haven't yet" from this and the one below. Also, I would avoid taking people they have to install home brew. Just link them to the mac version of ffmpeg and then maybe put a note in parentheses that you can use home brew